### ISSUE TITLE
[BUG]: accessibility checker identified problems

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [NOAA-FIMS/noaa-fims.github.io#36]
**Describe the bug**
There are a few accessibility issues that were identified using https://wave.webaim.org/report#/noaa-fims.github.io. You can also view the downloadable report (the `index.html` file in the zip file that downloads) from the [accessibility checker GitHub action](https://github.com/NOAA-FIMS/noaa-fims.github.io/actions/workflows/accessibility_audit.yml) to view what needs to be addressed. 

**To Reproduce**
Navigate to the link above.

**Expected behavior**
We should be passing with a score of 10 not 8.5.

**Screenshots**

<img width="355" height="440" alt="Image" src="https://github.com/user-attachments/assets/e29805ac-1ecb-4e22-b239-293fdc1b66f3" />

**Additional context**

**Comment by e-perl-NOAA** [comment]
There is now an [accessibility checker github action](https://github.com/NOAA-FIMS/noaa-fims.github.io/blob/main/.github/workflows/accessibility_audit.yml), as well as [accessibility copilot instructions](https://github.com/NOAA-FIMS/noaa-fims.github.io/blob/main/.github/instructions/accessibility.instructions.md) that have been added. The audit is failing right now of course but it does make progress towards being more *aware* of accessibility issues and creating pipelines that consider them. Anyone that would like to work on making progress towards this can view [past workflow run reports](https://github.com/NOAA-FIMS/noaa-fims.github.io/actions/workflows/accessibility_audit.yml) and if one is not available you can click the `Run workflow` button to re-run the workflow to generate one.
